src/parallel/DomainDecompMPIBase.h

```diff
@@ -249,6 +252,12 @@ class DomainDecompMPIBase: public DomainDecompBase {
 	MPI_Comm _comm;
 
 	NeighbourCommunicationScheme* _neighbourCommunicationScheme;
```

Comment:
```suggestion
	NeighbourCommunicationScheme* _neighborCommunicationScheme;
```
lol BE

Reply:
in ls1 it's the complete mix ;)

Reply:
so at some point this has to be cleaned up, but not now ;)